src/player.js

```diff
@@ -346,8 +346,6 @@ export default class Player extends FakeEventTarget {
       }
       if (this._selectEngineByPriority()) {
         this._appendEngineEl();
-        this._posterManager.setSrc(this._config.metadata.poster);
```

Comment:
You want to set the source anyway, whether if it will be hidden or showing later.